integration_tests/__tests__/snapshot-test.js

```diff
@@ -41,4 +49,74 @@ describe('Snapshot', () => {
       output['snapshot is not influenced by previous counter 0']
     ).not.toBe(undefined);
   });
+
+  describe('validates existance of snapshots correctly', () => {
+    const pathToOriginal = path.resolve(
+      __dirname,
+      '../snapshot/__tests__/snapshot.js'
+    );
+    const originalContent = String(fs.readFileSync(pathToOriginal));
```

Comment:
I should probably pass 'utf-8' as the encoding so that it returns a string instead of a buffer as I need it to be a string to make some change in the test, for example when I disable the first `toMatchSnapshot` by replacing the first call to it with `.not.toBe(undefined)`